package.json

```diff
@@ -65,6 +65,7 @@
 		"clsx": "2.1.1",
 		"cmdk": "^1.0.0",
 		"cva": "1.0.0-beta.1",
+		"decimal.js": "^10.4.3",
```

Comment:
Question @satococoa 

Was it difficult to achieve this with `big.js`?  
From the following points of view, I think it might be better to use `big.js` if there are no issues:

- Actively maintained  
- Smaller file size  

https://npmtrends.com/big.js-vs-bignumber.js-vs-decimal.js-vs-mathjs  

<img width="1308" alt="image" src="https://github.com/user-attachments/assets/c60650a3-6483-4ddb-bbca-a28598375272">

Reply:
@shige 

I chose decimal.js because it includes type definitions, whereas big.js doesn't (it needs `@types/big.js`).
However, big.js (with @types/big.js) would also work if the type definitions are good enough.

Reply:
@satococoa 
I understand the background!  
For now, I think it's fine as it is. 👌